.github/workflows/verify.yml

```diff
@@ -0,0 +1,50 @@
+name: Verify
+on:
+  push:
+    branches:
+      - main 
+  pull_request:
+    branches:
+      - main 
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    environment: verify
+
+    steps:
+      - name: Check if VITE_ANVIL_FORK_URL is set
+        run: |
+          if [ -z "${VITE_ANVIL_FORK_URL}" ]; then
+            echo "VITE_ANVIL_FORK_URL is NOT set."
+          else
+            echo "VITE_ANVIL_FORK_URL is set."
+            echo "First 5 characters: ${VITE_ANVIL_FORK_URL:0:15}"
+          fi
+        env:
+          VITE_ANVIL_FORK_URL: ${{ secrets.VITE_ANVIL_FORK_URL }}
+          
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up pnpm
+        uses: pnpm/action-setup@v2
```

Comment:
I thought this repo used yarn?

Reply:
oh i see you migrated to pnpm in this pr

Reply:
haha yeah, @roninjin10 what do you think? Thought maybe worth it for slight speedup